.github/ISSUE_TEMPLATE/kubernetes_bump.md

```diff
@@ -58,10 +51,25 @@ changes should be cherry-picked to all release series that will support the new
 * [ ] Update book:
   * Update supported versions in `versions.md`
   * Prior art: #9161
+
 * [ ] Issues specific to the Kubernetes minor release:
   * Sometimes there are adjustments that we have to make in Cluster API to be able to support
     a new Kubernetes minor version. Please add these issues here when they are identified.
 
+### Bump quickstart and kind image references in CAPD
```

Comment:
Should we add something about when this step can be performed (after the K8s version is GA + when there is a kind image for the new K8s version)